docs/pages/machine-id/deployment.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Deploy Machine ID
+description: Explains how to deploy Machine ID on your platform and join it to your Teleport cluster.
+tocDepth: 3
+---
+
+The first step to set up Machine ID is to deploy the `tbot` binary and join a
+Machine ID bot to your Teleport cluster. You can run the `tbot` binary on a
+number of platforms, from AWS and GitHub Actions to a generic Linux server or
+Kubernetes cluster. This guide shows you how to deploy Machine ID on your
+infrastructure.
+
+## Choosing a deployment method
+
+There are two considerations to make when determing how to deploy Machine ID on
+your infrastructure.
+
+### Your infrastructure
+
+The `tbot` binary runs as a container or on a Linux virtual machine. If you run
+`tbot` on GitHub Actions, you can use a ready-made [GitHub Actions
+workflow](https://github.com/marketplace/actions/teleport-auth).
+
+### Initial authentication method
+
+Machine ID authenticates itself with your Teleport cluster using one of the
+following methods:
+
+- **Static join token:** Your Teleport client tool generates a string and stores
+  it on the Teleport Auth Service. Machine ID provides this string,
+  demonstrating to the Auth Service that it belongs in your Teleport cluster.
+- **Platform-signed document:** The platform that hosts `tbot`, such as a
+  Kubernetes cluster or Amazon EC2 instance, provides a signed identity document
+  that Teleport can verify using the platform's certificate authority. 
```

Comment:
Should we emphasize that this is the preferred approach, as it's more secure (no shared secrets) and doesn't suffer from the one-time-use issues associated with static tokens?

Reply:
Done. To keep this brief within the format of the list, I only mentioned the security consideration.